src/compas_occ/geometry/curves/nurbs.py

```diff
@@ -601,10 +602,27 @@ def closest_point(self, point: Point, return_parameter: bool = False) -> Union[P
             The nearest as (point, parameter) tuple, if ``return_parameter`` is true.
         """
         projector = GeomAPI_ProjectPointOnCurve(point.to_occ(), self.occ_curve)
-        point = Point.from_occ(projector.NearestPoint())
+        try:
+            point = Point.from_occ(projector.NearestPoint())
+            if return_parameter:
+                parameter = projector.LowerDistanceParameter()
+
+        except RuntimeError:
```

Comment:
that is a very general error. can we be more specific?

Reply:
> 

where should this note about the modified behaviour go? Inside the docstring?

Reply:
yes

Reply:
ok